plugins/synaptics-prometheus/fu-synaprom-config.c

```diff
@@ -103,6 +104,13 @@ fu_synaprom_config_setup (FuDevice *device, GError **error)
 		 self->configid1, self->configid2,
 		 GUINT16_FROM_LE(cfg.version));
 
+	/* append the configid to the generated GUID */
+	devid = g_strdup_printf ("USB\\VID_%04X&PID_%04X&CFG1_%u&CFG2_%u",
```

Comment:
Ohh! Silly me, on more thing -- sorry. This needs to be documented in README.md in the same directory as this source file - with that LGTM, thanks.